docker/docker-compose.yml

```diff
@@ -10,8 +10,14 @@ services:
       - POSTGRES_PASSWORD=${POSTGRES_PASSWORD}
       - POSTGRES_DBNAME=n8n
     volumes:
-      - ./postgresql_init_data:/docker-entrypoint-initdb.d
-      - ./postgresql_data:/var/lib/postgresql/data:Z
+      - ./n8n-postgres/postgresql_init_data:/docker-entrypoint-initdb.d
+      - ./n8n-postgres/postgresql_data:/var/lib/postgresql/data:Z
+  rabbitmq:
```

Comment:
This repo was meant to hold only the n8n relevant things (so please remove rabbitmq and worker containers). 
Please create a new repository which only holds a docker-compose file which combines all the different components:
  * rabbitmq
  * n8n
  * needed databases
  * worker images

That way we can keep things separated

Reply:
Moved the components to [klips-project/rabbitmq-n8n](https://github.com/klips-project/rabbitmq-n8n).